### PR TITLE
Create lib-openblas-0.2.20-universal package

### PR DESCRIPTION
**Pull request by dsavenko**
Hi,

This PR covers https://github.com/ctuning/ck-math/issues/10 .

I tested the build on various systems, seems to work fine.

### CODE REVIEW

File: package/lib-openblas-0.2.20-universal/README.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+We do not compile Fortran API and LAPACK by default.
+
+To customize build use the following vars:
+
+* OPENBLAS_TARGET (str) to force target
+* OPENBLAS_FORTRAN (YES) to build Fortran API
+* OPENBLAS_LAPACK (YES) to build lapack
+
+For example,
+$ ck install package:lib-openblas-0.2.19-universal --env.OPENBLAS_TARGET=ARMV7 --env.OPENBLAS_FORTRAN=YES --env.OPENBLAS_LAPACK=YES
```
Comment (psyhtest) [inline package/lib-openblas-0.2.20-universal/README.txt]: 0.2.20

**Review by psyhtest — CHANGES_REQUESTED** [review]
1) Please exclude the directory package/lib-openblas-0.2.20-universal/arc from this commit.

2) Please use in headers:

See CK LICENSE.txt for licensing details.
See CK COPYRIGHT.txt for copyright details.

3) Please README.txt to README.md and update 0.2.19 to 0.2.20.

**Comment by psyhtest** [comment]
@dsavenko Sorry for the niggles. Just better to clean up now than again see them when copying `lib-openblas-0.2.20-universal` to ` lib-openblas-0.2.21-universal`, etc.

**Comment by dsavenko** [comment]
@psyhtest, done 1 and 3. I didn't get what you mean by 2. Those lines are already present in `info.json` and all the scripts. Please, clarify. 

> @dsavenko Sorry for the niggles. Just better to clean up now than again see them when copying lib-openblas-0.2.20-universal to lib-openblas-0.2.21-universal, etc.

It's OK, I totally understand. Should've been more careful.

**Comment by psyhtest** [comment]
>  I didn't get what you mean by 2. 

Just uniform capitalisations and file extensions everywhere e.g. `COPYRIGHT.txt` instead of `Copyright`.

**Review by psyhtest — APPROVED** [review]

**Comment by dsavenko** [comment]
Fixed capitalization and merged.

**Pull request merged**